instrumentation/clickhouse/clickhouse-client-0.8/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("com.clickhouse")
+    module.set("client-v2")
```

Comment:
i think it might be a little confusing for this instrumentation to have the same name as the other. perhaps the module could be `clickhouse-clientv2-0.8`?